components/NavigationDrawer/src/MDCBottomDrawerViewController.m

```diff
@@ -191,10 +195,19 @@ - (void)setElevation:(MDCShadowElevation)elevation {
   if ([self.presentationController isKindOfClass:[MDCBottomDrawerPresentationController class]]) {
     MDCBottomDrawerPresentationController *bottomDrawerPresentationController =
         (MDCBottomDrawerPresentationController *)self.presentationController;
+    BOOL elevationDidChange =
+        !MDCCGFloatEqual(bottomDrawerPresentationController.elevation, elevation);
```

Comment:
It's probably cleaner to put the following elevation assignment and call to `mdc_elevationDidChange` inside of `if (!MDCCGFloatEqual(bottomDrawerPresentationController.elevation, elevation)` – we don't need to assign if the value hasn't changed.

Reply:
done.